modules/Microsoft.Compute/virtualMachines/deploy.bicep

```
@@ -276,6 +276,16 @@ param provisionVMAgent bool = true
 @description('Optional. Indicates whether Automatic Updates is enabled for the Windows virtual machine. Default value is true. For virtual machine scale sets, this property can be updated and updates will take effect on OS reprovisioning.')
 param enableAutomaticUpdates bool = true
 
+@description('Optional. VM guest patching orchestration mode. Allowed values are \'AutomaticByPlatform\', \'AutomaticByOS (Windows only)\', \'Manual (Windows only)\', \'ImageDefault (Linux only)\'. Default value is blank (not set).')
```

```suggestion
@description('Optional. VM guest patching orchestration mode.')
```

All the other pieces of information are automatically injected into the ReadMe :)

---

I'd agree with @AlexanderSehr in general, but in this case we're losing information on which value is specific for which OS, since the readme will only list the available options.

---

I've updated the description again, this time added a link to the MS doc so users can refer to it when utilizing this parameter.